geolocation-API/PositionOptions.https.html

```diff
@@ -43,71 +43,50 @@
 
   try {
     geo.getCurrentPosition(
-        t86.step_func(function(pos) {
-          assert_unreached('A success callback was invoked unexpectedly');
-        }),
-        t86.step_func(function(err) {
+        t86.unreached_func('A success callback was invoked unexpectedly'),
+        t86.step_func_done(function(err) {
           assert_equals(err.code, err.TIMEOUT);
-          t86.done();
         }),
         {timeout: 0, maximumAge: 0}
     );
   } catch(e) {
-    t86.step(function() {
-      assert_unreached('An exception was thrown unexpectedly: ' + e.message);
-    });
+    t86.unreached_func('An exception was thrown unexpectedly: ' + e.message);
```

Comment:
I don't believe this works the way you want it to work. unreached_func() merely returns a function that when called fails the test, but you're not actually calling the function. Maybe rather than the `try {} catch (e) {}` you could just wrap the entire thing in  `t86.step(function() {...});`, which would automatically deal with unexpected exceptions being thrown?